src/searching.md

```diff
@@ -183,148 +182,151 @@ Searches that ignore combining characters are slower than regular searches.
+(<https://docs.rs/regex/1.3.9/regex/#syntax>)
 
 ## Card state
 
 `is:due`\
-review cards and learning cards waiting to be studied
+review cards and learning cards waiting to be studied.
 
 `is:new`\
-new cards
+new cards.
 
 `is:learn`\
-cards in learning
+cards in learning.
 
 `is:review`\
-reviews (both due and not due) and lapsed cards
+reviews (both due and not due) and lapsed cards.
 
 `is:suspended`\
-cards that have been manually suspended
+cards that have been automatically or manually suspended.
 
 `is:buried`\
 cards that have been buried, either [automatically](studying.md#siblings-and-burying) or
-manually
+manually.
 
-Note that with the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
+Note that with the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html),
 Anki now distinguishes between manually and automatically buried cards so you can
 unbury one set without the other.
 
 Cards that have lapsed fall into several of these categories, so it may
 be useful to combine them to get more precise results:
 
 `is:learn is:review`\
-cards that have lapsed and are awaiting relearning
+cards that have lapsed and are awaiting relearning.
 
 `-is:learn is:review`\
-review cards, not including lapsed cards
+review cards, not including lapsed cards.
 
 `is:learn -is:review`\
-cards that are in learning for the first time
+cards that are in learning for the first time.
 
 `flag:1`\
-cards with a red flag
+cards with a red flag.
 
 `flag:2`\
-cards with an orange flag
+cards with an orange flag.
 
 `flag:3`\
-cards with a green flag
+cards with a green flag.
 
 `flag:4`\
-cards with a blue flag
+cards with a blue flag.
 
 `flag:5`\
-cards with a pink flag
+cards with a pink flag.
 
 `flag:6`\
-cards with a turquoise flag
+cards with a turquoise flag.
 
 `flag:7`\
-cards with a purple flag
+cards with a purple flag.
 
 ## Card properties
 
 `prop:ivl>=10`\
-cards with interval of 10 days or more
+cards with interval of 10 days or more.
 
 `prop:due=1`\
-cards due tomorrow
+cards due tomorrow.
 
 `prop:due=-1`\
-cards due yesterday that haven’t been answered yet
+cards due yesterday that haven’t been answered yet.
 
 `prop:due>-1 prop:due<1`\
-cards due between yesterday and tomorrow
+cards due between yesterday and tomorrow.
 
 `prop:reps<10`\
-cards that have been answered less than 10 times
+cards that have been answered less than 10 times.
 
 `prop:lapses>3`\
-cards that have moved into relearning more than 3 times
+cards that have moved into relearning more than 3 times.
 
 `prop:ease!=2.5`\
-cards easier or harder than default
+cards easier or harder than default.
 
-`prop:cdn:d>5` (Requires Anki 2.1.64+.)\
-cards with the value of `d` in custom data (usually refers to Difficulty in FSRS) greater than 5
+`prop:cdn:d>5`\
+cards with the value of `d` in custom data (usually refers to difficulty in FSRS) greater than 5 (requires Anki 2.1.64+).
```

Comment:
I think capitalizing "Difficulty" is correct here, since it's referring to one of the titled values of FSRS.

Reply:
We don't capitalise DSR in the statistics screen of app. Here is the part of Sherlock's essay where he doesn't capitalise these terms: [Spaced Repetition Algorithm](https://github.com/open-spaced-repetition/fsrs4anki/wiki/Spaced-Repetition-Algorithm:-A-Three%E2%80%90Day-Journey-from-Novice-to-Expert#:~:text=Below%20are%20some%20empirical%20observations%20from%20Wozniak%2C%20corroborated%20by%20data%20from%20language%20learning%20platforms%20like%20MaiMemo%3A)